.travis.yml

```diff
@@ -9,24 +9,78 @@ branches:
 
 matrix:
   include:
-    - python: 3.7
+    - os: linux
+      python: 3.7
       env: NOX_SESSION=check
-    - python: 3.7
+    - os: linux
+      python: 3.7
       env: NOX_SESSION=docs
 
-    - python: 3.6
+    - os: linux
+      python: 3.6
       env: NOX_SESSION=test-3.6
-    - python: 3.7
+    - os: linux
+      python: 3.7
       env: NOX_SESSION=test-3.7
-    - python: 3.8-dev
+
+    - os: linux
+      python: 3.8-dev
       env: NOX_SESSION=test-3.8
       dist: bionic # Required to get OpenSSL 1.1.1+
 
+    - os: windows
+      language: shell
+      python: 3.7
+      env:
+        PATH=/c/Python37:/c/Python37/Scripts:$PATH
+        NOX_SESSION=check
+      before_install:
+        - choco install python --version 3.7
+        - python -m pip install --upgrade pip
+    - os: windows
+      language: shell
+      python: 3.7
+      env:
+        PATH=/c/Python37:/c/Python37/Scripts:$PATH
+        NOX_SESSION=docs
+      before_install:
+        - choco install python --version 3.7
+        - python -m pip install --upgrade pip
+
+    - os: windows
+      language: shell
+      python: 3.6
+      env:
+        PATH=/c/Python36:/c/Python36/Scripts:$PATH
+        NOX_SESSION=test-3.6
+      before_install:
+        - choco install python --version 3.6
+        - python -m pip install --upgrade pip
+    - os: windows
+      language: shell
+      python: 3.7
+      env:
+        PATH=/c/Python37:/c/Python37/Scripts:$PATH
+        NOX_SESSION=test-3.7
+      before_install:
+        - choco install python --version 3.7
+        - python -m pip install --upgrade pip
+
+    - os: windows
+      language: shell
+      python: 3.8-dev
+      env:
+        PATH=/c/Python38:/c/Python38/Scripts:$PATH
+        NOX_SESSION=test-3.8
+      before_install:
+        - choco install python --pre
+        - python -m pip install --upgrade pip
```

Comment:
Could we restrict these to only the `test-3.7` session? Check and docs aren't really dependent on the OS, and not running 3.6 and 3.8-dev is a good compromise to keeping the build light enough while having basic Windows coverage.